lib/screens/v2/widget_instance/reconstructed_alert.ex

```diff
@@ -1165,12 +1217,25 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlert do
 
   def serialize(widget, log_fn \\ &Logger.warning/1)
 
-  def serialize(%__MODULE__{is_full_screen: true, alert: %Alert{effect: effect}} = t, log_fn) do
-    diagram_data = serialize_diagram(t, log_fn)
+  def serialize(
+        %__MODULE__{
+          is_full_screen: true,
+          alert: %Alert{effect: effect} = alert,
+          all_platforms_at_informed_station: all_platforms_at_informed_station
+        } = t,
+        log_fn
+      ) do
+    location = LocalizedAlert.location(t)
 
-    main_data = pick_layout_serializer(t, diagram_data, effect, dual_screen_alert?(t))
+    if Alert.is_partial_station_closure?(alert, all_platforms_at_informed_station) do
+      t |> serialize_single_screen_fallback_alert(location)
```

Comment:
Pipeline not needed here for a single function call. (I keep wondering why Credo isn't complaining about this on various projects, so I finally looked it up — turns out it's tagged as "controversial" and thus opt-in. Why this is "controversial", I'm not sure.)

Reply:
Hmm, maybe we should opt-in. Any time I commit these, it's almost always on accident. Just the aftermath of a logic change where I didn't need the other pipes anymore.